pom.xml

```diff
@@ -47,9 +47,9 @@
 	<properties>
 		<maven.compiler.source>1.8</maven.compiler.source>
 		<maven.compiler.target>1.8</maven.compiler.target>
-		<spring-cloud-commons.version>2.2.1.BUILD-SNAPSHOT</spring-cloud-commons.version>
+		<spring-cloud-commons.version>2.2.2.BUILD-SNAPSHOT</spring-cloud-commons.version>
```

Comment:
Is there a Spring Cloud BOM we can use for all spring-cloud dependencies? Bus, Sleuth, Stream, Commons...

Reply:
Since we're included in Spring Cloud BOM, we can't use Spring Cloud BOM ourselves.